src/order_cmd.cpp

```diff
@@ -40,12 +41,15 @@
 #include "tracerestrict.h"
 #include "train.h"
 #include "date_func.h"
+#include "3rdparty/nlohmann/json.hpp"
+#include "command_aux.h"
 
 #include "table/strings.h"
 
 #include "3rdparty/robin_hood/robin_hood.h"
-
+#include <3rdparty/nlohmann/json.hpp>
```

Comment:
This include is duplicated, and in general system header syntax should not be used for local includes.